pkg/accumulator/accumulator_interface.go

```diff
@@ -0,0 +1,8 @@
+package accumulator
+
+type AcccumulatorClient interface {
+	// this function StartCacheAccumulator will store the data from the ebpf engine
```

Comment:
[Question] "store the data" this is contradicting the function name. Where does it store the data? Or that by "store" you mean "cache", so don't use different words, it is confusing.

Reply:
done